examples/lighting-app/bouffalolab/common/AppTask.cpp

```diff
@@ -159,8 +152,17 @@ void AppTask::AppTaskMain(void * pvParameter)
     ef_get_env_blob(APP_REBOOT_RESET_COUNT_KEY, &resetCnt, sizeof(resetCnt), &saved_value_len);
     resetCnt++;
     ef_set_env_blob(APP_REBOOT_RESET_COUNT_KEY, &resetCnt, sizeof(resetCnt));
+
+    GetAppTask().mButtonPressedTime = chip::System::SystemClock().GetMonotonicMilliseconds64().count() + 1;
```

Comment:
Please document why the +1 is there.  Yes, I know you just moved this code.

Reply:
Add comments for this.